src/AWS.Deploy.CLI/ConsoleInteractiveServiceImpl.cs

```diff
@@ -7,6 +7,11 @@ namespace AWS.Deploy.CLI
 {
     public class ConsoleInteractiveServiceImpl : IToolInteractiveService
     {
+        public ConsoleInteractiveServiceImpl()
+        {
+            Console.Title = "AWS .NET Deployment Tool";
```

Comment:
String literal should be in constants shared project.

Reply:
Fixed